packages/xod-arduino-deploy/src/boardsParser.js

```diff
@@ -42,19 +54,44 @@ const loadBoards = R.curry(
 //
 // =============================================================================
 
+// :: FQBN -> Map boardIdentifier BoardPrefs -> BoardPrefs
+const pickBoardPrefsByFqbn = R.curry(
```

Comment:
I’d name it `getBoardPrefsByFqbn` because `pick` is already exploited by Ramda with a different semantics